service/controller/v1/resource/patchfinalizer/create.go

```diff
@@ -0,0 +1,92 @@
+package patchfinalizer
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	finalizerPCC = "operatorkit.giantswarm.io/prometheus-config-controller"
+	labelCluster = "giantswarm.io/cluster"
```

Comment:
Is this label only used for CP services?

Reply:
Yes.